src/software/interface_service.sh

```
@@ -0,0 +1,154 @@
+# service.sh
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)